backend/docker_ops.py

```
@@ -5,7 +5,7 @@
 from typing import List, Dict, Optional, Tuple
 import asyncio
 from pathlib import Path
-from models import VolumeMount
+from .models import VolumeMount
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)